### PR TITLE
Service dashboard bug

### PR DESCRIPTION
**Pull request by paulgheorghecristian**
Right now, when adding a host with the same port, address and username as another, the existing one gets overridden because the hashmap in the ServerManager class containing the hosts uses (port + : + address + : + username) as the key. 

The problem is that it gets overridden only in the second list, so host "a" still remains in the list to the left.

It must be that the person who coded this mechanism didn't want to have this kind of duplicates, so now I'm checking whether such a duplicate exist and, if it does, I don't allow the user to create a host.

Also, this update eliminates altogether the hashmap and uses the list of hosts in the CMHostNodePersistManager because the hashmap is of no use.

Another update in this PR is that now we differentiate between "connect host" and "edit host" action. When clicking the connect host button, the fields are no longer editable.

### CODE REVIEW

File: com.cubrid.cubridmanager.core/src/com/cubrid/cubridmanager/core/common/ServerManager.java [hunk 1]
```diff
@@ -28,14 +28,13 @@
...
 package com.cubrid.cubridmanager.core.common;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Map;
 
+import com.cubrid.common.ui.spi.model.CubridServer;
```
Comment (seunghun-kim) [inline com.cubrid.cubridmanager.core/src/com/cubrid/cubridmanager/core/common/ServerManager.java]: Package `com.cubrid.common.ui` dependent the package `com.cubrid.cubridmanager.ui`.
So, as you added package `com.cubrid.common.ui` in here cause cyclic dependency.

Would you please fix this problem?

**Comment by newpcraft** [comment]
@paulgheorghecristian 
It seems these unnecessary classes should have to be removed, so you might remove them.

By the way, I'd like to touch your code due to the cyclic dependency issue. However, now there is no way to put my additional code in your pull request. So, I will create a new pull request based on your latest changes. Please check out my changes and let me know your suggestion.

**Comment by seunghun-kim** [comment]
@kisoo-han It should be closed because merged in #65. 😄

**Comment by kisoo-han** [comment]
* Close outdated PR

**Pull request closed without merging**